aptos-move/aptos-gas-schedule/src/gas_schedule/misc.rs

```diff
@@ -90,6 +93,7 @@ where
     V: ValueVisitor,
 {
     deref_visitor_delegate_simple!(
+        [visit_native_value, SizedID],
```

Comment:
Is `SizedID` the new name for the native handle? Why is it called "sized"?

Reply:
Sized because it also carries size, might not be a good name, but in any case I am trying to merge this with `DelayedFieldID` because these are essentially the same.

Reply:
Dumb question. Just wondering what it means to carry size. Does it just mean that we compute the size of SizedID? Or does it mean that SizedID is of the format (size, id)?

Reply:
It is. pair of ID and size of the serialised value it replaced. Btw, do you have suggestions for names? If this is confusing we probably should change it

Reply:
Gotcha. Now, it makes sense.

Reply:
@vusirikala ended up removing `SizedID` completely, just reused `DelayedFieldID` because as you pointed out it is indeed confusing!